schema-engine/core/src/commands/diff_cli.rs

```diff
@@ -129,13 +130,19 @@ fn namespaces_and_preview_features_from_diff_targets(
 // `None` in case the target is empty
 async fn json_rpc_diff_target_to_dialect(
     target: &DiffTarget,
+    datasource_urls: &ValidatedDatasourceUrls,
     shadow_database_url: Option<&str>, // TODO: delete the parameter
```

Comment:
This is still used by the `--shadow-database-url` CLI flag which can be different from the `shadowDatabaseUrl` as defined in the config file. And I think it makes sense to keep it because you probably don't want to add `shadowDatabaseUrl` to the config file just for the sake of a one-off command to diff against a migration directory if you don't need a custom shadow db otherwise.